recipes/srwpy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "srwpy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Synchrotron-Radiation-Workshop/SRW/archive/v{{ version }}.tar.gz
+  sha256: 53bdcb0c02c59659bd0cb3849eac15c166db65b69d6fb62d4b8bf97cc4c96be8
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - fftw  # [win]
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - fftw  # [win]
+    - matplotlib-base
+    - numpy
+    - pillow
+    - scipy
+
+test:
+  imports:
+    - srwpy
+    - srwpy.srwl_bl
+    - srwpy.srwlib
+    - srwpy.srwlpy
+    - srwpy.uti_math
+
+about:
+  home: https://github.com/Synchrotron-Radiation-Workshop/SRW
+  license: BSD-3-Clause
```

Comment:
The license doesn't look like BSD-3-Clause?

Reply:
@chrisburr sorry! My mistake. Any recommendation on what to add to license family for something so specific?

Reply:
Looks like a custom license. I did not read thoroughly to figure out if we can re-distribute it or not but if we can it would be `LicenseRef-SRW`.

Reply:
@ocefpaf we can redistribute as long as the license is available. I will change the license_family and add a new maintainer to the list as well. Thank you for your review.

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/14021/commits/102cc9decf4f755717b6c4bd7be681931e2f3805